sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/AbstractMethodParameterLookup.cs

```diff
@@ -72,11 +72,16 @@ public bool TryGetParameterSymbol(TArgumentSyntax argument, out IParameterSymbol
 
         public bool TryGetSymbolParameter(IParameterSymbol parameter, out TArgumentSyntax argument)
         {
-            foreach(var Pair in GetAllArgumentParameterMappings())
+            return TryGetArgumentSyntax(parameter.Name, out argument);
+        }
+
+        public bool TryGetArgumentSyntax(string parameterName, out TArgumentSyntax argument)
```

Comment:
Do we consider this tested since all cases are covered by `TryGetSymbolParameter(IParameterSymbol parameter...` ?

Reply:
I would say yes for that case, as `TryGetSymbolParameter(IParameterSymbol parameter...` is a wrapper around `TryGetSymbolParameter(string parameterName...`, and explicitely retesting the wrapped method would clutter the test files in my opinion, as it would be almost copy-pasted code testing similar test examples